QUEST.tsv

```diff
@@ -2249,31 +2249,31 @@ QUEST_20150717_002252	What do you think? They make a pretty good friend, right?
 QUEST_20150717_002253	Item Merchant 
 QUEST_20150717_002254	I'm sorry I can't come forward myself. {nl}But the demons will sense it when I use the spell.
 QUEST_20150717_002255	I will fix the twisted spell of the tower.{nl} I will have to use the spell at first and monsters will rush in. Please protect me.
-QUEST_20150717_002256	I hope Gabija is safe… Let's move fast.
+QUEST_20150717_002256	$I hope Gabija's safe… Let's move fast.
```

Comment:
I tend to agree. Contractions might make sentences more compact, but we shouldn't make things too colloquial.